backend/src/main/java/ca/bc/gov/restapi/results/postgres/endpoint/UserRecentOpeningEndpoint.java

```diff
@@ -0,0 +1,51 @@
+package ca.bc.gov.restapi.results.postgres.endpoint;
+
+import ca.bc.gov.restapi.results.common.pagination.PaginatedResult;
+import ca.bc.gov.restapi.results.oracle.dto.OpeningSearchResponseDto;
+import ca.bc.gov.restapi.results.postgres.service.UserRecentOpeningService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/openings/recent")
+public class UserRecentOpeningEndpoint {
+
+  private final UserRecentOpeningService userRecentOpeningService;
+
+  /**
+   * Retrieves a list of recent openings viewed by the user, limited by the number of results.
+   *
+   * @param limit The maximum number of results to return.
+   * @return A list of opening IDs viewed by the user.
+   */
+  @GetMapping
+  public ResponseEntity<PaginatedResult<OpeningSearchResponseDto>> getUserRecentOpenings(
```

Comment:
This endpoint was defined but it is never called by the frontend. Either remove it from the backend or use it on the frontend.

Reply:
removed this endpoint